### PR TITLE
chore(add-replica): add replica monitoring

### PR DESCRIPTION
**Pull request by Cali93**
### Context

Adds monitoring for the Supabase read replica following this [reference](https://github.com/supabase/supabase-grafana?tab=readme-ov-file#read-replica-support).

Note: [This project](https://github.com/supabase/grafana-agent-fly-example/tree/main?tab=readme-ov-file) has been deprecated in favour of https://github.com/supabase/supabase-grafana so we might also migrate to the latest solution which includes more metrics.

### CODE REVIEW

File: agent.yaml [hunk 1]
```diff
@@ -19,6 +19,18 @@ metrics:
...
+          scrape_interval: 30s
+          scheme: https
+          metrics_path: "/customer/v1/privileged/metrics"
+          basic_auth:
+            username: service_role
+            password: ${SUPABASESERVICEROLESECRET}
+          static_configs:
+            - targets: ["awyuaxlpezyhfkpznrox-rr-eu-central-1-tuoer.supabase.co"]
+              labels:
+                - supabase_project_ref: "awyuaxlpezyhfkpznrox-rr-eu-central-1-tuoer"
```
Comment (chris13524) [inline agent.yaml:33]: Can we use variables e.g. `${SUPABASEPROJECTURL}` for some of these? Otherwise lgtm

**Review by chris13524 — APPROVED** [review]

File: agent.yaml (hunk 1 above)
Comment (Cali93) [reply agent.yaml:33]: I agree it'd be better but I don't have access to the repo secrets and this is not a secret anyway.
Any chance you have admin access on this one ?

Comment (chris13524) [reply agent.yaml:33]: I do not have access, but someone else does and is an easy add

Looks like they are part of the `infra` repo though:
- https://github.com/WalletConnect/infra/blob/3c2d7dab2d1db1c20eb7f8d4a5fd67ae8ecd6153/terraform/supabase_metrics/main.tf#L47
- https://github.com/WalletConnect/infra/blob/3c2d7dab2d1db1c20eb7f8d4a5fd67ae8ecd6153/.github/workflows/cd.yml#L13

If not secret then not a big deal I suppose.

**Review by arein — APPROVED** [review]

**Comment by bkrem** [comment]
@Cali93 think we can close this since the read replica monitoring was set up in another PR with Nodar right?

**Comment by Cali93** [comment]
Yes indeed, it was taken care of by https://github.com/WalletConnect/infra/pull/102

**Pull request closed without merging**